pkg/controller/backup_sync_controller.go

```diff
@@ -34,24 +36,24 @@ import (
 	listers "github.com/heptio/ark/pkg/generated/listers/ark/v1"
 	"github.com/heptio/ark/pkg/persistence"
 	"github.com/heptio/ark/pkg/plugin"
-	"github.com/heptio/ark/pkg/util/kube"
 	"github.com/heptio/ark/pkg/util/stringslice"
 )
 
 type backupSyncController struct {
 	*genericController
 
-	client                      arkv1client.BackupsGetter
-	backupLister                listers.BackupLister
-	backupStorageLocationLister listers.BackupStorageLocationLister
-	namespace                   string
+	backupClient                arkv1client.BackupInterface
+	backupLocationClient        arkv1client.BackupStorageLocationInterface
+	backupLister                listers.BackupNamespaceLister
+	backupStorageLocationLister listers.BackupStorageLocationNamespaceLister
```

Comment:
Would it still make sense if we removed `backup` from the name of these vars?

Reply:
given that we have clients and listers for multiple API objects here, I'd prefer to explicitly name them